### PR TITLE
French translation and other little things

### PR DESCRIPTION
**Pull request by jendib**

### CODE REVIEW

File: grails-app/assets/javascripts/streama/translations/EN_us.js [hunk 1]
```diff
@@ -34,14 +34,14 @@ angular.module('streama.translations').config(function ($translateProvider) {
...
 			GENRE: 'Genre',
 			TRAILER: 'Trailer',
-			SEASON: 'Season',
+			SEASON: 'Season'
 		},
 
 		MESSAGES: {
 			SHARE_SOCKET: 'By creating a new session you will be redirected back to this player, but this time you will have a unique session ID in the url. Share this with your friends to have a syncronized watching experience with them!',
-			FILE_MISSING: 'There is a problem with this content. It seems you removed the associated video file from it.. Share this with your friends to have a syncronized watching experience with them!',
+			FILE_MISSING: 'There is a problem with this content. It seems you removed the associated video file from it.',
```
Comment (dularion) [inline grails-app/assets/javascripts/streama/translations/EN_us.js]: Why did you remove that last sentence?

File: grails-app/conf/application.groovy [hunk 2]
```diff
@@ -2,6 +2,7 @@ grails.databinding.dateFormats = [
     "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", // javascript format in json
     "yyyy-MM-dd HH:mm:ss.S",
     "yyyy-MM-dd'T'hh:mm:ss'Z'",
+    "yyyy-MM-dd'T'hh:mm:ssZ",
```
Comment (dularion) [inline grails-app/conf/application.groovy]: Thanks for this! Good catch

File: grails-app/conf/application.yml [hunk 3]
```diff
@@ -20,6 +20,11 @@ environments:
             url: jdbc:mysql://localhost/streama
             username: root
             password:
+    development-no-mysql:
+            dataSource:
+                dbCreate: update
+                driverClassName:  'org.h2.Driver'
```
Comment (dularion) [inline grails-app/conf/application.yml]: Interesting choice, although for easier debugging I actually use mysql locally. Maybe each contributor should set this as desired without pushing?

File: grails-app/assets/javascripts/streama/translations/EN_us.js (hunk 1 above)
Comment (jendib) [inline grails-app/assets/javascripts/streama/translations/EN_us.js]: This last sentence isn't a mistake? Why would we want to share the fact that a file is missing with our friends?

File: grails-app/conf/application.yml (hunk 3 above)
Comment (jendib) [inline grails-app/conf/application.yml]: It's still a choice for developers. If I don't push it, it's a risk having conflicts if you or someone else commit something here.

File: grails-app/assets/javascripts/streama/translations/EN_us.js (hunk 1 above)
Comment (dularion) [inline grails-app/assets/javascripts/streama/translations/EN_us.js]: Oh oops, I thought it was the message about the synchronized playback.. Haha you're right of course

File: grails-app/conf/application.yml (hunk 3 above)
Comment (dularion) [inline grails-app/conf/application.yml]: All right all right :)
